### DISCUSSION TITLE
Information on knowledgebase features

### DISCUSSION THREAD

**Discussion by henricook** [outline/outline#1787] [Self Hosting]
Hi all,

Outline looks like a really cool Wiki. I can't find a page that focuses on its knowledgebase features though, is there one?

I want to build a Q&A database and have things like synonyms for the questions so that I can describe all the different ways that people ask the question to get to the same answer

**Comment by matbrgz** [comment]
I like the Q&A feature, this could be an internal plugin working like an embed.

**Comment by almereyda** [comment]
This sounds similar to what https://www.documize.com/ does.

Yet I feel there are better options for real Q&A platforms, where user feedback can be integrated directly.

Which are the kind of *knowledge base features* you were considering here @henricook ? From what I can grasp, you are thinking of some kind of interactive FAQ. What else comes up to your mind when thinking *knowledge base*?

**Reply by henricook** [reply]
Thanks for asking, in my extremely narrow-view case I'm looking for a system that helps people who answer our security questionnaires internally. e.g. We'll get asked 5 differently worded variations of "Do you make sure users use long passwords?" or "Do all your laptops use full disk encryption?" - what would be really great is if the people completing these forms can search for canned answers to these questions and after a minimal check just copy and paste into the (inevitably) spreadsheet questionnaire.

So to my mind that's like Q&A system, with some kind of 'synonym search' for the questions, or relevancy matching (scoring keywords).
